en/docs/reference/connectors/file-connector/file-connector-config.md

```diff
@@ -2143,6 +2244,36 @@ The following operations allow you to work with the File Connector version 4. Cl
             </td>
         </tr>
     </table>
+    
+    **Sample configuration**
+    ```xml
+    <file.read configKey="CONNECTION_NAME">
+        <path>{$ctx:path}</path>
+        <filePattern>{$ctx:filePattern}</filePattern>
+        <includeResultTo>{$ctx:includeResultTo}</includeResultTo>
+        <resultPropertyName>{$ctx:resultPropertyName}</resultPropertyName>
+        <readMode>{$ctx:readMode}</readMode>
+        <startLineNum>{$ctx:startLineNum}</startLineNum>
+        <endLineNum>{$ctx:endLineNum}</endLineNum>
+        <lineNum>{$ctx:lineNum}</lineNum>
+        <contentType>{$ctx:contentType}</contentType>
+        <encoding>{$ctx:encoding}</encoding>
+        <enableStreaming>{$ctx:enableStreaming}</enableStreaming>
+        <enableLock>{$ctx:enableLock}</enableLock>
+        <maxRetries>{$ctx:maxRetries}</maxRetries>
+        <retryDelay>{$ctx:retryDelay}</retryDelay>
+    </file.read>
+    ```
+    
+    **Sample request**
+
+    Following is a sample REST/JSON request that can be handled by the append operation.
+    ```json
+        {
+            "path":"/home/vive/Desktop/file/read.txt",
+            "readMode":"Complete File"
```

Comment:
Shouldn't this be Reads complete file

Reply:
No, `Complete File` is the correct value. I fixed this incorrect value in other places too.